### PR TITLE
Fix glyphs

### PR DESCRIPTION
**Pull request by Jped**

### CODE REVIEW

File: src/parsettx.py [hunk 1]
```diff
@@ -4,12 +4,11 @@
...
-    'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'X', 'Y', 'Z', 'a', 'b', 'c', 'd', 'e',
+    'P', 'Q', 'R', 'S', 'T', 'U', 'V','W', 'X', 'Y', 'Z', 'a', 'b', 'c', 'd', 'e',
     'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't',
     'u', 'v', 'w', 'x', 'y', 'z', 'zero', 'one', 'two', 'three', 'four',
-    'five', 'six', 'seven', 'eight', 'nine', 'space', 'exclam', 'quotedbl',
-    'numbersign', 'dollar', 'percent', 'ampersand', 'quotesingle', 'parenleft',
-    'parenright', 'asterisk', 'plus', 'comma', 'hyphen', 'period', 'slash',
+    'five', 'six', 'seven', 'eight', 'nine', 'space', 'exclam', 
+    'numbersign', 'dollar', 'percent', 'ampersand', 'asterisk', 'comma', 'hyphen', 'period', 'slash',
     'colon', 'semicolon', 'less', 'equal', 'greater', 'question', 'at',
```
Comment (eigenfoo) [inline src/parsettx.py]: I thought we agreed to remove `less`, `greater` and `equal`? And also some brackets and the `asciicircum`. @ostapstephan @ccurro can you please confirm

Comment (ccurro) [reply src/parsettx.py]: All you mentioned are the be removed. Also hyphen, slash, comma, colon, semicolon, period. All the brackets

Comment (ccurro) [reply src/parsettx.py]: Bar, underscore, grave too

Comment (ostapstephan) [reply src/parsettx.py]: these should be included:
'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M', 'N',
        'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z', 'a', 'b', 'c',
        'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q',
        'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z', 'zero', 'one', 'two',
        'three', 'four', 'five', 'six', 'seven', 'eight', 'nine', 
        'exclam', 'numbersign', 'dollar', 'percent', 'ampersand',
        'asterisk','question', 'at',

Comment (Jped) [reply src/parsettx.py]: I guess, I did not change it .... lol ill push the new one soon

**Pull request merged**